python/ray/serve/backend_state.py

```diff
@@ -91,11 +94,20 @@ def __set_state__(self, d: Dict[Any, Any]) -> None:
         self._ready_obj_ref = None
         self._drain_obj_ref = None
 
+    @property
+    def replica_tag(self) -> str:
+        return str(self._replica_tag)
+
+    @property
+    def backend_tag(self) -> str:
+        return self._backend_tag
+
     @property
     def actor_handle(self) -> ActorHandle:
-        return ray.get_actor(self._actor_name)
+        return ray.get_actor(
+            self._actor_name, namespace=self._controller_namespace)
 
-    def start_or_update(self, backend_info: BackendInfo):
+    def _handle_placement_group(self, backend_info):
```

Comment:
this is a pretty non-descriptive name. is there any reason we should be handling the placement group creation separate from the actor creation logic? I'd find it more intuitive if they were handled in the same place.

Reply:
Btw, where do we recover the placement group in the case that we're recovering from controller failure?

Reply:
This part of logic was the first section for `start_or_update` that creates a placement group gated by a flag and pass it down as arguments to actor handle. I just happened to move this part to a helper function for isolation.

For update and recover -- we didn't recreate actor handle in update cases in current implementation, since they have actor handle already with placement group passed in we can simply left them out.